gen-apidocs/generators/html.go

```diff
@@ -617,16 +617,16 @@ func (h *HTMLWriter) generateHTML(navContent string) {
 	fmt.Fprintf(html, "%s", string(buf))
 	fmt.Fprintf(html, "</DIV>\n</DIV>\n")
 	fmt.Fprintf(html, "<SCRIPT src=\"/js/jquery-3.2.1.min.js\"></SCRIPT>\n")
-	fmt.Fprintf(html, "<SCRIPT src=\"jquery.scrollTo.min.js\"></SCRIPT>\n")
+	fmt.Fprintf(html, "<SCRIPT src=\"js/jquery.scrollTo.min.js\"></SCRIPT>\n")
 	fmt.Fprintf(html, "<SCRIPT src=\"/js/bootstrap-4.3.1.min.js\"></SCRIPT>\n")
```

Comment:
@kbhawkey Note the way bootstra-4.3.1.min.js is referenced. The path has a prefix '/'. That is how we reference the existing script in website

Reply:
Changes seem fine now. Agreed?

Reply:
Yes. Per the intent of this PR, it is fine now. There are still many places we can iterate and improve in future PRs.

Reply:
👍